### ISSUE TITLE
Why mse not binary_crossentropy

### ISSUE THREAD

**Issue by david30907d** [eriklindernoren/Keras-GAN#45]
[cycelgan.py](https://github.com/eriklindernoren/Keras-GAN/blob/master/cyclegan/cyclegan.py#L50)

Hi @eriklindernoren , thanks for your repo

I really learned a lot. But something confuses me, why you use mse as loss function for discriminator not cross entropy?

I thought that cross entropy would give much bigger gradient, isn't that helpful for training?

**Comment by gspell** [comment]
Soooo can we reopen this, as it never received an answer?
